client/rest-high-level/src/test/java/org/elasticsearch/client/security/AuthenticateResponseTests.java

```diff
@@ -70,12 +70,16 @@ protected AuthenticateResponse createTestInstance() {
         final String fullName = randomFrom(random(), null, randomAlphaOfLengthBetween(0, 4));
```

Comment:
Since you touched the response, I am doing my duty to recommend moving these tests to `AbstractHlrcStreamableXContentTestCase` .

Reply:
I will address this in a followup PR